apps/coordinator/src/components/ScriptExplorer/UTXOSet.jsx

```diff
@@ -170,13 +173,49 @@ class UTXOSet extends React.Component {
     }
   };
 
+  getChipProperties(amountSats, feeRate, walletConfig) {
+    const wasteMetrics = new WasteMetrics();
+    const scriptType = walletConfig.addressType;
```

Comment:
if you're doing this in the component, there's no need to pass as function arguments since these are available on the props.